plugin/evm/service.go

```diff
@@ -86,7 +86,7 @@ type VersionReply struct {
 }
 
 // ClientVersion returns the version of the VM running
-func (service *AvaxAPI) Version(r *http.Request, args *struct{}, reply *VersionReply) error {
+func (service *AvaxAPI) Version(r *http.Request, _ *struct{}, reply *VersionReply) error {
```

Comment:
q: why didn't linter catch this?

Reply:
I don't think coreth uses `revive` (which is what would catch this in avalanchego)